dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -429,6 +466,29 @@ public void manualPublish(final List<String> args) {
         }
     }
 
+    private DockstoreTool.RegistryEnum getRegistryEnum(String registry) {
+        for (DockstoreTool.RegistryEnum reg : DockstoreTool.RegistryEnum.values()) {
+            if (registry.equals(reg.name())) {
+                return reg;
+            }
+        }
+        return null;
```

Comment:
Let's change this into an `Optional` return or an exception. I think it isn't clear above what happens if an unknown registry is entered.